src/core/swap/swap-api.js

```diff
@@ -94,7 +95,7 @@ export async function fetchSwapQuote(
   if (promises.length < 1) throw new Error('No swap providers enabled')
 
   // Wait for the results, with error handling:
-  return fuzzyTimeout(promises, 20000).then(
+  return fuzzyTimeout(promises, 120000).then(
```

Comment:
Do we want 2 minutes? Remember, `fuzzyTimeout` will wait as long as it needs for at least one success to come back (even if that takes forever). This timeout is for situations where we already have successful results, but there are still a few pending promises. In that case, `fuzzyTimeout` will return what it has so far and ditch the stragglers.

Reply:
ack. That snuck in as it was for debugging. will remove